packages/e2e-tests/tests/inspector-elements-03_nested-node-selection.test.ts

```diff
@@ -0,0 +1,114 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+import { openDevToolsTab, startTest } from "../helpers";
+import { openConsolePanel, warpToMessage } from "../helpers/console-panel";
+import {
+  getElementsPanelSelection,
+  getElementsRowWithText,
+  getElementsTree,
+  openAppliedRulesTab,
+  openElementsPanel,
+  searchElementsPanel,
+  selectElementsRowWithText,
+  typeKeyAndVerifySelectedElement,
+  waitForElementsToLoad,
+  waitForSelectedElementsRow,
+} from "../helpers/elements-panel";
+import {
+  stackingTestCases,
+  verifyStackingTestCaseSelectedElementUnderCursor,
+} from "../helpers/elements-panel";
+import { toggleToolboxLayout } from "../helpers/layout";
```

Comment:
nit A bunch of these imports are unused (Opt+Shift+O will remove)

(This is true for all for all of the new test files fwiw)